homeassistant/components/filter/sensor.py

```diff
@@ -79,7 +79,7 @@
 ICON = "mdi:chart-line-variant"
 
 FILTER_SCHEMA = vol.Schema(
-    {vol.Optional(CONF_FILTER_PRECISION, default=DEFAULT_PRECISION): vol.Coerce(int)}
+    {vol.Optional(CONF_FILTER_PRECISION, default=None): vol.Any(None, vol.Coerce(int))}
```

Comment:
Definition of `DEFAULT_PRECISION` can be removed.

I don't understand how removing default precision only applies to 3 filters, while all filter schemas are extending from `FILTER_SCHEMA`? If we want a different default value, we could keep track of a dictionary mapping `filter name => default value`, only used when we detect that precision is None.